builder/openstack/access_config.go

```diff
@@ -273,3 +288,127 @@ func (c *AccessConfig) getEndpointType() gophercloud.Availability {
 	}
 	return gophercloud.AvailabilityPublic
 }
+
+type DebugRoundTripper struct {
+	ui                packer.Ui
+	rt                http.RoundTripper
+	numReauthAttempts int
+}
+
+// RoundTrip performs a round-trip HTTP request and logs relevant information about it.
+func (drt *DebugRoundTripper) RoundTrip(request *http.Request) (*http.Response, error) {
+	defer func() {
+		if request.Body != nil {
+			request.Body.Close()
+		}
+	}()
+
+	var err error
+
+	drt.ui.Message(fmt.Sprintf("Request Method: %s", request.Method))
+
+	if request.Body != nil {
+		request.Body, err = drt.logRequestBody(request.Body, request.Header)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	info, err := json.MarshalIndent(request.Header, "", "  ")
+	if err != nil {
+		drt.ui.Message(fmt.Sprintf("Error logging request headers: %s\n", err))
```

Comment:
```suggestion
		drt.ui.Message(fmt.Sprintf("[DEBUG] Error logging request headers: %s\n", err))
```
I would add the [DEBUG] tag in front of these logs.